app/assets/stylesheets/application.scss

```diff
@@ -48,3 +48,9 @@
 .solicitor_results_list {
   padding-left: 14px;
 }
+
+[data-fn="btn-solicitor-results-close"] {
```

Comment:
Is a data attribute selector going to work in IE7? I don't know - just asking the question.